hudi-common/src/test/java/org/apache/hudi/common/table/log/HoodieLogFormatTest.java

```diff
@@ -78,6 +80,8 @@
 @RunWith(Parameterized.class)
 public class HoodieLogFormatTest {
 
+  @Rule
+  public TemporaryFolder folder = new TemporaryFolder();
```

Comment:
I am wondering if we should reuse @yanghua 's work here. May be create a thin HoodieCommonTestHarness class and make it extend the HoodieClientTestHarness and make these tests subclass HoodieCommonTestHarness?

Reply:
+1 for the solution about unifying resource management.

Reply:
Thanks for the comments. I will update the PR soon.

Reply:
go for it! :)